playwright-e2e/authentication/auth-dsm.ts

```diff
@@ -45,7 +45,7 @@ export async function login(page: Page, opts: { email?: string; password?: strin
     return;
   }
   // Retry login if login window is still present (POST /auth0 fails intermittenly)
-  console.log(`Retry log in`);
+  console.log(`      Retry log in`);
```

Comment:
should this be logInfo?

Reply:
used to help me to see retry works or not. I don't need it anymore, will remove.